packages/modelviewer.dev/examples/scenegraph/index.html

```diff
@@ -560,6 +560,111 @@ <h2 class="demo-title">Transform textures</h2>
     </div>
 
 
+    <div class="sample">
+      <div id="hideShowMeshVariantsExample" class="demo"></div>
+      <div class="content">
+        <div class="wrapper">
+          <div class="heading">
+            <h2 class="demo-title">Toggle between Mesh Variants</h2>
+            <p>If you have a few mesh variants, you can hide and show them by changing textures opacity.</p>
+          </div>
+          <example-snippet stamp-to="hideShowMeshVariantsExample" highlight-as="html">
+            <template>
+<model-viewer id="manifold" camera-controls touch-action="pan-y" src="../../shared-assets/models/manifold.glb" ar alt="A 3D model of a stick and ball">
+  <div class="controls">
+    <div>Mesh Variants: <select id="textures"></select>
+    </div>
+</model-viewer>
+<script type="module">
+const mvTextures = document.querySelector("model-viewer#manifold");
+const textures = document.querySelector('#textures');
+
+mvTextures.addEventListener("load", () => {
+  const materials = mvTextures.model.materials
+  for(const material of materials) {
+    const option = document.createElement('option')
+    option.value = material.name;
+    option.textContent = material.name;
+    textures.appendChild(option);
```

Comment:
If it's possible to do something declaratively (HTML) rather than procedurally (JS), always prefer HTML. Let's put these options in the `<div>` above.

Reply:
Thanks for the feedback. Updated the select to have the options declaratively.